src/powergslb/database/redis/timeseries.py

```diff
@@ -0,0 +1,82 @@
+import logging
+
+import redis
+from kairos import Timeseries
+
+__all__ = [ 'RedisTimeSeries' ]
+
+class RedisTimeSeries():
+
+  def __init__(self, **kwargs):
+        self.client = None
+        try:
+          self.client = redis.StrictRedis(host=kwargs['host'], port=kwargs['port'], db=kwargs['db'])
+          self.client.ping()
+          logging.debug('Redis host=%s,port=%s,db=%d- Connected!', kwargs['host'], kwargs['port'], kwargs['db'])
+        except Exception as ex:
+          self.client = None
+          logging.error("Redis host=%s,port=%s,db=%d- Error %s", ex, kwargs['host'], kwargs['port'], kwargs['db'])
+          pass
+
+        self.ts = None
+        if self.client != None:
+          logging.debug('Timeseries - Create')
+          if 'timeseries' in kwargs:
+            self.ts = Timeseries(self.client, type='gauge', intervals = kwargs['timeseries'] )
+          else:
+            self.ts = Timeseries(self.client, type='gauge', intervals = {
+                'seconds': {
+                  'step': 5, # 5 seconds
+                  'steps': 120, # last 10 minutes
+                  'read_cast' : float,
+                }
+              }
+            )
+
+  def record_hit( self, key, measurement ):
+    if self.client:
+      self.ts.insert( str(key), float(measurement) )
+
+  def record_response_time( self, content_id, measurement ):
+    self.record_hit( str(content_id)+':rt', float(measurement) )
```

Comment:
Here and below. Not consistent operations style (not single space around operation signs).